mage_integrations/mage_integrations/sources/couchbase/__init__.py

```diff
@@ -32,7 +32,10 @@ def build_connection(self) -> CouchbaseConnection:
             scope=self.config['scope'],
             connection_string=self.config['connection_string'],
             password=self.config['password'],
-            username=self.config['username']
+            username=self.config['username'],
+            cert_path=self.config['cert_path'],
```

Comment:
for optional fields, let's use `self.config.get('cert_path')`. Same for the two fields below.